http-transport-binding.md

```diff
@@ -168,19 +168,6 @@ Examples:
     * `eventID` maps to `CE-EventID`
     * `cloudEventsVersion` maps to `CE-CloudEventsVersion`
 
-For the `extensions` attribute, each entry of the `extensions` map
```

Comment:
I thought each transport binding would also indicate a default placement for extensions. Is that not the case or are you waiting for Clemens to add?

Reply:
I was assuming Clemens would open a new PR to do that part of the work. He has an AI:
> AI Clemens: Need an “escape hatch” in our binding specs to allow attributes to override the serialization rules defined.